parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -82,6 +82,22 @@ downloadCredentalProvider() {
     retrycmd_get_tarball 120 5 "$CREDENTIAL_PROVIDER_DOWNLOAD_DIR/$CREDENTIAL_PROVIDER_TGZ_TMP" "$CREDENTIAL_PROVIDER_DOWNLOAD_URL" || exit $ERR_CREDENTIAL_PROVIDER_DOWNLOAD_TIMEOUT
 }
 
+downloadCredentalProviderWithOras() {
+    mkdir -p $CREDENTIAL_PROVIDER_DOWNLOAD_DIR
+    CREDENTIAL_PROVIDER_TGZ_TMP=${CREDENTIAL_PROVIDER_DOWNLOAD_URL##*/}
+    oras pull $CREDENTIAL_PROVIDER_DOWNLOAD_URL -o $CREDENTIAL_PROVIDER_TGZ_TMP || exit $ERR_CREDENTIAL_PROVIDER_DOWNLOAD_TIMEOUT
+}
+
+installOras() {
```

Comment:
why is this during cloud init and not during Packaker build ?

Reply:
note that we source cse_install.sh during builds though: https://github.com/Azure/AgentBaker/blob/04f13a59a305b8709a8d936a94dc8aa1143e77b2/vhdbuilder/packer/install-dependencies.sh#L17C1-L17C42

Reply:
I'd say it would definitely be worth it to install oras during the build process, especially if it's critical to downloading required dependencies at runtime